src/fabric/src/fabric_doc_open_revs.erl

```diff
@@ -327,19 +327,19 @@ collapse_duplicate_revs_int([Reply | Rest]) ->
 
 setup_all() ->
     config:start_link([]),
-    meck:new([fabric, couch_stats, couch_log]),
-    meck:new(fabric_util, [passthrough]),
-    meck:expect(fabric, update_docs, fun(_, _, _) -> {ok, nil} end),
-    meck:expect(couch_stats, increment_counter, fun(_) -> ok end),
-    meck:expect(couch_log, notice, fun(_, _) -> ok end),
-    meck:expect(fabric_util, cleanup, fun(_) -> ok end).
+    ok = meck:new([fabric, couch_stats, couch_log]),
+    ok = meck:new(fabric_util, [passthrough]),
+    ok = meck:expect(fabric, update_docs, fun(_, _, _) -> {ok, nil} end),
+    ok = meck:expect(couch_stats, increment_counter, fun(_) -> ok end),
+    ok = meck:expect(couch_log, notice, fun(_, _) -> ok end),
+    ok = meck:expect(fabric_util, cleanup, fun(_) -> ok end).
```

Comment:
I wouldn't bother with `ok =`, I think these only return `ok` or throw an error. This won't catch an error anyway.

Reply:
Removed all `ok = ` from `meck:expect()` and `meck:reset()` in `fabric` module, thanks for the review!